shoop/utils/iterables.py

```diff
@@ -29,11 +29,8 @@ def batch(iterable, count):
     """
     Yield batches of `count` items from the given iterable.
 
-    >>> for x in batch([1, 2, 3, 4, 5, 6, 7], 3):
-    >>>   print(x)
-    [1, 2, 3]
-    [4, 5, 6]
-    [7]
+    >>> tuple(x for x in batch([1, 2, 3, 4, 5, 6, 7], 3))
```

Comment:
OK, now I got it... My fix was a bit overkill. It could have been:
```python
    >>> for x in batch([1, 2, 3, 4, 5, 6, 7], 3):
    ...     print(x)
```

i.e. the continuation lines should be starting with `...` (also indendation should be 4, but that's not a syntax error).